Makefile

```diff
@@ -121,6 +120,9 @@ clean: clean_docs
 	rm -rf $(RELEASE_DIR)
        
 
+$(HOST_BUILD_DIR)/integration.test: $(SOURCES)
```

Comment:
What is the use of this target, may be put that details to commit log also?

Reply:
I forgot to add it but we should check we can build integration test on all platforms. Perhaps in a new target like `make cross-integration`.

Reply:
@guillaumerose but then crc binary for mac/windows not going to run on linux so `make cross-integration` will not work on a single platform.

Reply:
I added the CI target, let me know what you think of https://github.com/code-ready/crc/pull/1437/commits/fa753f28b43688cf69ac09b78a107726bddff89e !